styles.css

```diff
@@ -940,6 +940,11 @@ div.database-plugin__td.data-input span:focus {
 *               React-datepicker styles
 ************************************************************/
 /** VARIABLES */
+
+.react-datepicker__aria-live {
+  display: none !important;
```

Comment:
!important should be avoided as a good practice . If you thinks there is no alternative I will aprove the PR

Reply:
sadly it's not working, is it possible that this doesn't get bundled when compiling or something like that?

Reply:
the css its a mess. I am starting to learn sass with the other plugin to be more scalable and maintainable.

With the next minor version I will just add the skeleton and required dependencies to use it. We can go here with !importat tag then